.github/workflows/integration_bugtracker.yml

```diff
@@ -48,6 +48,7 @@ jobs:
         export GH_BUGTRACKER_INTEGRATION_TEST_API_TOKEN="${{ secrets.GH_BUGTRACKER_INTEGRATION_TEST_API_TOKEN }}"
         export JIRA_BUGTRACKER_INTEGRATION_API_USERNAME="${{ secrets.JIRA_BUGTRACKER_INTEGRATION_API_USERNAME }}"
         export JIRA_BUGTRACKER_INTEGRATION_API_TOKEN="${{ secrets.JIRA_BUGTRACKER_INTEGRATION_API_TOKEN }}"
+        export AZURE_BOARDS_BUGTRACKER_INTEGRATION_API_TOKEN="${{ secrets.AZURE_BUGTRACKER_INTEGRATION_API_TOKEN }}"
```

Comment:
Name is `AZURE_BOARDS_INTEGRATION_API_TOKEN`, see https://github.com/kiwitcms/Kiwi/pull/2034#discussion_r512247366